snap/snapcraft.yaml

```diff
@@ -191,8 +191,8 @@ parts:
       install -T bundles/dynbinary-daemon/dockerd "$CRAFT_PART_INSTALL/bin/dockerd"
       # install docker-proxy previously provided by libnetwork part
       install -T bundles/dynbinary-daemon/docker-proxy "$CRAFT_PART_INSTALL/bin/docker-proxy"
-    # https://github.com/moby/moby/blob/v24.0.5/Dockerfile.simple (Docker-supported Go version for Engine)
-    build-snaps: &go ['go/1.20/stable']
+    # https://github.com/moby/moby/blob/v27.2.0/Dockerfile.simple (Docker-supported Go version for Engine)
```

Comment:
Should this be based on Dockerfile.simple or Dockerfile? 
In any case, better link to the right line: https://github.com/moby/moby/blob/v27.2.0/Dockerfile#L3